python/ray/data/read_api.py

```diff
@@ -2122,42 +2094,27 @@ def from_huggingface(
             )
 
     Args:
-        dataset: A `Hugging Face Datasets Dataset`_ or `DatasetDict`_.
-            :class:`~ray.data.IterableDataset` isn't supported.
+        dataset: A `Hugging Face Datasets Dataset`_.
+            :class:`~ray.data.IterableDataset` and
```

Comment:
Man this IterableDataset link is just straight up wrong 😅. Not sure how this had slipped in before.